collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -11,36 +11,47 @@
 disabled_by_default = True
 
 
-def charts_template(sm):
+def charts_template(sm, alarm_status_chart_type='line'):
     order = [
         'alarms',
+        'values'
     ]
 
     mappings = ', '.join(['{0}={1}'.format(k, v) for k, v in sm.items()])
     charts = {
         'alarms': {
-            'options': [None, 'Alarms ({0})'.format(mappings), 'status', 'alarms', 'alarms.status', 'line'],
+            'options': [None, 'Alarms ({0})'.format(mappings), 'status', 'alarms', 'alarms.status', alarm_status_chart_type],
             'lines': [],
             'variables': [
                 ['alarms_num'],
             ]
+        },
+        'values': {
+            'options': [None, 'Alarm Values', 'value', 'values', 'alarms.values', 'line'],
```

Comment:
```suggestion
            'options': [None, 'Alarm Values', 'value', 'value', 'alarms.values', 'line'],
```

Singular seems better to me. What do you think?

Reply:
if plural is better, then we need to make both families plural: statuses, values.